### PR TITLE
Update to Swift 3

### PR DESCRIPTION
**Pull request by onmyway133**
- This updates the framework and example projects to Swift 3
- Remove Quick as test target dependency

### CODE REVIEW

File: SwiftPackageTests/Shared/SharedTests.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    // Use XCTAssert and related functions to verify your tests produce the correct results.
+  }
+
+  func testPerformanceExample() {
+    // This is an example of a performance test case.
+    self.measure {
+      // Put the code you want to measure the time of here.
+    }
+  }
+
```
Comment (RamonGilabert) [inline SwiftPackageTests/Shared/SharedTests.swift:26]: You can delete this space.

File: SwiftPackageTests/Mac/macOSTests.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    // Use XCTAssert and related functions to verify your tests produce the correct results.
+  }
+
+  func testPerformanceExample() {
+    // This is an example of a performance test case.
+    self.measure {
+      // Put the code you want to measure the time of here.
+    }
+  }
+
```
Comment (RamonGilabert) [inline SwiftPackageTests/Mac/macOSTests.swift:26]: This space can be gone.

File: SwiftPackageTests/iOS/iOSTests.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    // Use XCTAssert and related functions to verify your tests produce the correct results.
+  }
+
+  func testPerformanceExample() {
+    // This is an example of a performance test case.
+    self.measure {
+      // Put the code you want to measure the time of here.
+    }
+  }
+
```
Comment (RamonGilabert) [inline SwiftPackageTests/iOS/iOSTests.swift:26]: And this.

**Review by RamonGilabert — COMMENTED** [review]
Only spacing in tests.

**Comment by RamonGilabert** [comment]
Also, have you selected Use Legacy Version Swift to No in Xcode?

File: SwiftPackageTests/Shared/SharedTests.swift (hunk 1 above)
Comment (vadymmarkov) [reply SwiftPackageTests/Shared/SharedTests.swift:26]: @RamonGilabert Normally we do have a line in the end of the file.

File: SwiftPackageTests/Shared/SharedSpec.swift [hunk 4]
```diff
@@ -1,16 +0,0 @@
-@testable import SwiftPackage
-import Quick
-import Nimble
-
-class SharedSpec: QuickSpec {
```
Comment (vadymmarkov) [inline SwiftPackageTests/Shared/SharedSpec.swift:5]: 😢

File: SwiftPackageTests/Shared/SharedTests.swift (hunk 1 above)
Comment (onmyway133) [reply SwiftPackageTests/Shared/SharedTests.swift:26]: @zenangst can Marvin on Xcode 8 still automatically add this new line at the end?

File: SwiftPackageTests/Shared/SharedSpec.swift (hunk 4 above)
Comment (onmyway133) [reply SwiftPackageTests/Shared/SharedSpec.swift:5]: @vadymmarkov should we make `Quick` optional? 😉

**Comment by onmyway133** [comment]
Is this good to merge? 😉  @hyperoslo/ios

**Comment by onmyway133** [comment]
🚀

**Pull request merged**